src/NzbDrone.Core/Indexers/Definitions/SpeedCD.cs

```diff
@@ -19,14 +18,13 @@
 using NzbDrone.Core.Messaging.Events;
 using NzbDrone.Core.Parser;
 using NzbDrone.Core.Parser.Model;
-using NzbDrone.Core.Validation;
 
 namespace NzbDrone.Core.Indexers.Definitions
 {
-    public class SpeedCD : TorrentIndexerBase<UserPassTorrentBaseSettings>
+    public class SpeedCD : TorrentIndexerBase<SpeedCDSettings>
```

Comment:
This will need a DB migration since it's changing the ConfigContract for the Indexer. All else looks good here.

Reply:
Yep, that works